packages/@ember/application/tests/dependency_injection_test.js

```diff
@@ -13,7 +13,12 @@ moduleFor(
     constructor() {
       super();
 
-      application = run(EmberApplication, 'create');
+      // Must use default resolver because test resolver does not normalize
```

Comment:
Gotcha, so lets split the tests that rely on the globals resolver into their own moduleFor + class (e.g. `moduleFor('Application Dependency Injection - Globals Resolver [DEPRECATED]', class extends TestCase {})`).

Reply:
There's only one test that relies on it, so I'm just going to move it into `default_resolver_test.js`.